README.md

```diff
@@ -141,6 +143,14 @@ redis-server --port 8082 &
 
 Once you have started the server, open your preferred web browser and navigate to <http://127.0.0.1:8080> to start using Websurfx.
 
+## Using docker
+
+If you are using docker you can quickly start the project for development purpose using docker compose.
+
+```sh
+docker compose up
+```
+
```

Comment:
I would like to suggest a minor change here, can you add this section in the [docs](https://github.com/neon-mmd/websurfx/tree/rolling/docs) with a new doc page dedicated to this and also explaining in depth about the process of setting docker deployment for development purposes. Moreover, if there is a need to add notes like `warning` or `notes` then please add them as well.

Here is an example of a dedicated `developing` doc page from the `dashy` project in their docs:

https://github.com/Lissy93/dashy/blob/master/docs/developing.md

Reply:
I feel like this is more your responsability to write the developing guide as you have more knowledge about the project then i have.

Reply:
I think you are right, no problem, we will update for these changes soon :slightly_smiling_face: .